src/modules/FunModule.cpp

```diff
@@ -539,6 +576,20 @@ FunModule::FunModule(UmikoBot* client) : Module("funutil", true), m_memeChannel(
 
 	});
 
+	RegisterCommand(Commands::FUN_GITHUB, "github", [this](Client& client, const Message& message, const Channel& channel) 
+	{
+		QStringList args = message.content().split(' ');
+
```

Comment:
This variable is used nowhere, you can check for the correct input and make this variable used.
like
`if(args.size() != 1) { /*send message*/ return; }`